app/models/manageiq/providers/openstack/network_manager/security_group.rb

```diff
@@ -1,2 +1,85 @@
 class ManageIQ::Providers::Openstack::NetworkManager::SecurityGroup < ::SecurityGroup
+  include ProviderObjectMixin
+  include AsyncDeleteMixin
+  include SupportsFeatureMixin
+
+  supports :create_security_group
+  supports :delete_security_group
+  supports :update_security_group
+
+  def self.raw_create_security_group(ext_management_system, options)
+    cloud_tenant = options.delete(:cloud_tenant)
+    security_group = nil
+    ext_management_system.with_provider_connection(connection_options(cloud_tenant)) do |service|
+      security_group = service.create_security_group(options).body
+    end
+    {:ems_ref => security_group['id'], :name => options[:name]}
+  rescue => e
+    _log.error "security_group=[#{options[:name]}], error: #{e}"
+    raise MiqException::MiqSecurityGroupCreateError, e.to_s, e.backtrace
+  end
+
+  def raw_delete_security_group
+    ext_management_system.with_provider_connection(connection_options(cloud_tenant)) do |service|
+      service.delete_security_group(ems_ref)
+    end
+  rescue => e
+    _log.error "security_group=[#{name}], error: #{e}"
+    raise MiqException::MiqSecurityGroupDeleteError, e.to_s, e.backtrace
+  end
+
+  def delete_security_group_queue(userid)
+    task_opts = {
+      :action => "deleting Security Group for user #{userid}",
+      :userid => userid
+    }
+    queue_opts = {
+      :class_name  => self.class.name,
+      :method_name => 'delete_security_group',
```

Comment:
@ladas, sorry, yes, delete_security_group and update_security_group, should be both prefixed with raw because they're actually calling the raw API commands. Fixed it.